docs/tutorials/execution.md

```diff
@@ -24,10 +24,32 @@ The default task runner is the `ConcurrentTaskRunner`, which will run submitted
 
 All Prefect task runners support asynchronous task execution.
 
+## Result
+By default, the result of a task is a Python object, which blocks the execution of the next task in a flow. To make sure that the tasks within your flow can run concurrently or in parallel, add `.submit()` to your task run. This method will return a `PrefectFuture` instead of a Python object.
```

Comment:
```suggestion
By default, the result of a task is a Python object, and execution of the task blocks the execution of the next task in a flow. To make sure that the tasks within your flow can run concurrently or in parallel, add `.submit()` to your task run. This method will return a `PrefectFuture` instead of a Python object.
```